home/styles.css

```diff
@@ -30,7 +30,10 @@ main {
   width: 100%;
   height: 10vh;
```

Comment:
why in height in vh ? asking because i guess we use vh to cover whole screen but if some container is small enough to give 10vh then why not use rem instead ?

Reply:
i want it to be as fixed as its for nav